nestkernel/synaptic_element.cpp

```diff
@@ -120,7 +120,10 @@ nest::SynapticElement::set( const DictionaryDatum& d )
   double new_tau_vacant = tau_vacant_;
 
   // Store values
-  updateValue< double >( d, names::growth_rate, growth_rate_ );
+  if ( d->known( names::growth_rate ) )
```

Comment:
I agree with @apeyser than this looks weird.
Also, this might be a stupid question since I'm not completely familiar with how NEST deals with structural plasticity, yet, but is it clear that all synaptic elements share the same ``growth_rate`` value?
If not, wouldn't the current ``set`` make all of them equal in [``Archiving_Node::set_status``](https://github.com/sdiazpier/nest-simulator/blob/325ec678099cb251735793b28e14499b710ca216/nestkernel/archiving_node.cpp#L288)?